layers/+lang/idris/packages.el

```diff
@@ -84,4 +85,19 @@
         "sN" 'spacemacs/idris-load-forward-line-and-focus
         "sp" 'idris-load-backward-line
         "sP" 'spacemacs/idris-load-backward-line-and-focus
-        "ss" 'idris-pop-to-repl))))
+        "ss" 'idris-pop-to-repl)))
+
+  ;; open special buffers in motion state so they can be closed with ~q~
+  (evil-set-initial-state 'idris-compiler-notes-mode 'motion)
+  (evil-set-initial-state 'idris-hole-list-mode 'motion)
+  (evil-set-initial-state 'idris-info-mode 'motion))
```

Comment:
Perhaps because they can be closed with `C-g`, but yes for the sake of consistency they should have `q`.